gravitee-apim-cypress/cypress/integration/rest-api/apis/import-create-test.ts

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER } from '../../../fixtures/fakers/users/users';
+import { deleteApi, importCreateApi, getApiById } from '../../../commands/management/api-management-commands';
+import { getPages, getPage } from '../../../commands/management/api-pages-management-commands';
+import { ApiImportFakers } from '../../../fixtures/fakers/api-imports';
+import { ApiFakers } from '../../../fixtures/fakers/apis';
+
+context('API - Imports', () => {
+  describe('Create empty API without ID', function () {
```

Comment:
can you use arrow function instead of this please ? (applies everywhere)
`function () {` becomes `() => {`

Reply:
Yes, I changed the syntax everywhere :+1: